packages/tools/package/src/commands/changelog/changelog.types.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Changelog Command Options interface.
+ *
+ * @public
+ */
+export interface Options {
+  /**
+   * Packages updated as part of the version update.
+   *
+   * @remarks
+   * If no packages are defined, this will collect all workspace packages.
+   */
+  packages: Array<string>;
+
+  /**
+   * Tag to update the changelog for.
+   */
+  tag: string;
+
+  /**
+   * Previous commit id
+   */
+  commit: string;
+}
+interface CommitDetails {
+  /**
+   * Details about the commits added as part of the version.
+   */
+  [key: string]: string;
+}
+
+export interface AlongWithData {
```

Comment:
Q. What does this `AlongWithData` means?

Reply:
Details about the alongwithdata is given in line 33-36